### ISSUE TITLE
invalid method (createAttributedString) passed to UIModule

### ISSUE THREAD

**Issue by oakleaf** [FokkeZB/ti-html2as#25]
Hello, I use Ti SDK 5.0 with iOS9 & html2as v.1.3.2

In the simulator everything works perfect, but on the device, im getting the red Application error that says:

"invalid method (createAttributedString) passed to UIModule at nl.fokkezb.html2as.js (line 1)"

**Comment by FokkeZB** [comment]
Could you do me a favour and log `Ti.version` from in that same app to see what the value is?

https://github.com/FokkeZB/ti-html2as/blob/master/index.js#L4

**Comment by oakleaf** [comment]
Hey ofcourse!

Ti.version == 5.0.0

**Comment by FokkeZB** [comment]
What is puzzling me is that you shouldn't be able to use packaged CommonJS modules with 5.0.0.GA anyway because of https://jira.appcelerator.org/browse/TIMOB-19545. Or did build and drop in just the JS file?

Anyway, that's what I did to trace your bug and the problem was that because I used a reference to either {{Ti.UI}} or {{Ti.UI.iOS}} Titanium didn't include the class in device builds. I'll push a fix soon.
